### DISCUSSION TITLE
Sleeping mode and calibration mode MBP 14 M1 Max

### DISCUSSION THREAD

**Discussion by Franskaboi** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1214] [General]
I have some questions about my use of AlDente. Just went for the pro version and tried my first calibration which resulted in a fail as while I was discharging the battery I unplugged then plugged again which crashed the process.  

Also, I’m surprised as I can’t put my mbp in sleep mode while doing calibration process (which imo shouldn’t be a problem even w calibration) even when my computer is in the discharging phase. 
I don’t want my screen and my computer to stay on for 15 hours even when I don’t use it for the whole process of calibration. 

how can I deal with calibration without encountering that issue ? Am I doing something wrong ? 

Thanks in advance for your explanation,

Kind regards,

F

**Comment by MatthiasKerbl** [comment]
Hi @Franskaboi,

First of all, if you unplug your MacBook during the calibration process and plug it in again you will receive the following notification from AlDente:
<img width="372" alt="Screenshot 2024-04-22 at 10 11 04" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/33470003/e51868d6-c9bb-47ef-9774-d163d1e9dab0">
After clicking on "OK" the calibration will continue as intended.

Unfortunately, due to technical limitations, it is necessary to disable sleep during the whole calibration process. However, we are looking into doing something about the screen staying on.

If these limitations are something you don't like, you can do a manual calibration.

Best,
Matthias

**Reply by Franskaboi** [reply]
Thank you for your answer. I'm still a little struggling with my first steps with AlDente. Sadly it just canceled my calibration when I unplugged I'll do the test next week to see if I have this pop up (in the mean while I did manual calibration).
I'll let you know and can't wait for sleeping mode while calibrating. 
Thanks again,

Best,

F

**Reply by Franskaboi** [reply]
And sorry to bother one last time. Can you confirm me there is no way of stopping the charge while on sleep on M1s MacBooks ? This would be gold cause I'll be able to let it plugged all night. The gymnastic to plug it unplug it at every use is is kind of inconfortable.

Thanks,

F

**Reply by MatthiasKerbl** [reply]
Hi @Franskaboi,

Please check out the following articles about AlDente and Sleep:

- [Feature Explanation: Stop charging when sleeping](https://apphousekitchen.com/feature-explanation-stop-charging-when-sleeping/)
- [Feature Explanation: Disable Sleep until Charge Limit](https://apphousekitchen.com/feature-explanation-disable-sleep-until-charge-limit/)

Best,
Matthias

**Comment by iamonzon** [comment]
Hi, I am new to AlDente pro and I am doing the calibration process. If I unplug the computer while discharging phase would that be a problem? I know from docs and current experience it doesn't need to be unplug however I want to move and I can't without unplugging it. May be a redundant question, I just don't want to start the process over  😅

**Reply by iamonzon** [reply]
Quick update: I just unplugged it and nothing happened haha. I just asked perplexity and made me paranoid of it, yet it is safe (and it makes sense) to unplug it while discharging
![image](https://github.com/user-attachments/assets/e18169b3-9368-42cc-bb31-05c91dd77622)

Just one extra mention, not a question, during charging phase I had 2 displays connected and my computer never reached 100%. After unplug those I was able to get it 🙌
